cmd/utils/flags.go

```diff
@@ -954,6 +954,12 @@ Please note that --` + MetricsHTTPFlag.Name + ` must be set to start the server.
 		Value:    metrics.DefaultConfig.InfluxDBOrganization,
 		Category: flags.MetricsCategory,
 	}
+
+	CollectWitnessesFlagName = &cli.BoolFlag{
```

Comment:
This flag is just hanging dead now, isn't it? (I mean, it's not actually usable since it's not added to any cmd)

Reply:
Yup.  Pushed a fix for that.